snuba/datasets/discover.py

```diff
@@ -85,22 +86,54 @@ def get_data_source(self):
         return None
 
     def get_columns(self) -> ColumnSet:
-        common = ColumnSet([])  # TODO: Fill out
+        common = ColumnSet([
+            ('event_id', FixedString(32)),
+            ('project_id', UInt(64)),
+            ('timestamp', DateTime()),
```

Comment:
Will this map to `finish_ts` for transaction events?

Reply:
Yes, this is done in `column_expr`